books/architecture/src/foreword.md

```diff
@@ -0,0 +1,36 @@
+# Foreword
```

Comment:
Mostly inspired from [inspirational foreword in the Rust book](https://doc.rust-lang.org/book/foreword.html).

Too pretentious? Not pretentious enough? Perfect?!?

Reply:
Right on the edge :) 

I am ok with it though